backend/fms_core/services/samplesheet.py

```diff
@@ -218,6 +217,9 @@ def add_section_header(section_name: str):
 
     samplesheet.append([])
     add_section_header("BCLConvert_Settings")
+    ADAPTER_READ1S = { "NovaSeqXSeries": "CTGTCTCTTATACACATCT+ATGTGTATAAGAGACA" }
```

Comment:
I think these adapter sequences are linked to the adapter kit not the instrument type.